index.d.ts

```diff
@@ -79,6 +79,14 @@ export function useInput(
 	inputHandler: (input: string, key: Key) => void
 ): void;
 
+export function useFocus(): boolean;
+
+interface WithFocusProps {
+	hasFocus: boolean;
+}  
+
+export function withFocus(WrappedComponent: React.ComponentClass): React.ComponentClass<WithFocusProps>;
```

Comment:
I'm not really sure about this line, I am not really used to typescript :man_shrugging: